orm/connection.go

```diff
@@ -0,0 +1,71 @@
+package orm
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	_ "github.com/joho/godotenv/autoload"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var connection *gorm.DB
+var databaseName = "notifications"
+var Charset = "utf8mb4"
+var Collation = "utf8mb4_bin"
```

Comment:
I am more used to using a collation of `utf8mb4_unicode_ci`, but I have also seen people suggesting to use `utf8mb4_bin` so I guess it is more of a personal preference.  Also not sure what the better collation is these days as it has been a while since I checked on the latest recommendation so maybe `utf8mb4_bin` could be considered the better collation these days.

Reply:
Yeah, I just kind of picked one, I forget my exact reasoning. Since we are wanting a level of international use, `utf8mb4_0900_ai_ci` might be the best, provided our MySQL version is high enough. 

https://stackoverflow.com/a/71346996/2329474

Reply:
I think the one you mentioned is definitely a step up from the one I had. We can worry about other collations later on.

Reply:
Sure sounds good.  Yeah we can worry about perfecting the collation some more later on.